scripts/ocp_cleanup.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import sys
+
+from cfme.containers.provider.openshift import OpenshiftProvider
+from cfme.utils.providers import list_providers, ProviderFilter
+from cfme.utils.log import create_sublogger, add_stdout_handler
+
+logger = create_sublogger('ocp_cleanup')
+add_stdout_handler(logger)
```

Comment:
I think there's an issue with calling add_stdout_handler here, since create_sublogger returns a NamedLoggerAdapter:

```
Traceback (most recent call last):
  File "scripts/ocp_cleanup.py", line 9, in <module>
    add_stdout_handler(logger)
  File "/home/mshriver/repos/integration_tests/cfme/utils/log.py", line 531, in add_stdout_handler
    for handle in logger.handlers:
AttributeError: 'NamedLoggerAdapter' object has no attribute 'handlers'
```

Looks like you'll need to pass logger.logger, using the `logging.LoggerAdapter` attribute that stores the original logger object, to add_stdout_handler.

Reply:
I didn't expect that. fixed.

Reply:
I wouldn't have either, its a bit odd of behavior, but useful since the logger's an easily accessible attribute of the wrapper class.